### ISSUE TITLE
Can not `interact` with TPFs containing nan values

### ISSUE THREAD

**Issue by danhey** [KeplerGO/lightkurve#679]
#### Problem description

G'day all, I'm trying to `interact` with a target pixel file which contains some nan values at certain cadences. Every time I do this, I get thrown a bokeh error: `ValueError: Out of range float values are not JSON compliant`.

#### Example

```python
import lightkurve as lk
tpf = lk.search_targetpixelfile('TIC 99769182', mission='TESS').download_all()[0]
tpf.interact()
```

#### Expected behavior

I expect the TPF to be interactive with a cadence slider. I think this error is thrown because nan values can't be serialised with JSON. At the very least, we should catch this and raise a warning.

It is possible to plot the nan arrays in Bokeh:
```python
import numpy as np
from lightkurve.interact import prepare_tpf_datasource
from bokeh.plotting import figure, ColumnDataSource
from bokeh.io import show, output_notebook

tpf = lk.search_targetpixelfile('TIC 99769182', mission='TESS').download_all()[0]
pedestal = -np.nanmin(tpf.flux) + 1
tpf_source = prepare_tpf_datasource(tpf, tpf.pipeline_mask)
fig = figure(toolbar_location="below",
             border_fill_color="whitesmoke")
# TPF contains nans in last few entries:
fig.image([tpf.flux[-1, :, :] + pedestal],x=tpf.column, y=tpf.row,
              dw=tpf.shape[2], dh=tpf.shape[1], dilate=True,)
fig.rect('xx', 'yy', 1, 1, source=tpf_source, fill_color='gray',
                fill_alpha=0.4, line_color='white')
output_notebook(verbose=False, hide_banner=True)
show(fig)
```
#### Environment

-  platform (e.g. Linux, OSX, Windows): MacOS
-  lightkurve version (e.g. 1.0b6): 1.8.dev
-  installation method (e.g. pip, conda, source): source

**Comment by barentsen** [comment]
Thanks @danhey!  I am able to reproduce this bug. 

Looks like it is caused by the new `ylim_func` option, because the above example does work for me when I use:
```python
tpf.interact(ylim_func=lambda lc: (0, 300))
```

**Comment by barentsen** [comment]
I think I fixed this over in #679!

**Comment by danhey** [comment]
Cool, thanks @barentsen !!

**Comment by barentsen** [comment]
The fix has been merged!  I will likely release v1.7.1 soon to make the fix easy to install, but for now it should all work again on master!

Thanks for reporting the issue @danhey!! :+1: :+1:
